include/platform/radio.h

```diff
@@ -207,7 +262,8 @@ void otPlatRadioSetShortAddress(otInstance *aInstance, uint16_t aShortAddress);
  * @retval ::kThreadError_None     Successfully enabled.
  * @retval ::kThreadError_Failure  The radio could not be enabled.
  */
-ThreadError otPlatRadioEnable(otInstance *aInstance);
+ThreadError otPlatRadioEnable(otInstance *aInstance, otPlatRadioReceiveDone receiveCallback,
+                              otPlatRadioTransmitDone transmitCallback);
```

Comment:
If that is the consensus, I am fine with that. I'll give folks a bit more time to review before I go ahead and make these changes.